components/login/LoginForm.vue

```diff
@@ -206,7 +206,10 @@ async function loginUser() {
       >
         Sign in
       </button>
+<<<<<<< HEAD
```

Comment:
merge conflict issues remaining

Reply:
Removed it.  Hopefully I didn't miss any others.